src/components/FaqSection.tsx

```diff
@@ -0,0 +1,280 @@
+"use client";
+import { Box, Button, Typography, colors, styled } from "@mui/material";
+import ExpandMoreIcon from "@mui/icons-material/ExpandMore";
+import Link from "next/link";
+
+import MuiAccordion, { AccordionProps } from "@mui/material/Accordion";
+import MuiAccordionSummary, {
+  AccordionSummaryProps,
+} from "@mui/material/AccordionSummary";
+import MuiAccordionDetails from "@mui/material/AccordionDetails";
+
+const Accordion = styled((props: AccordionProps) => (
+  <MuiAccordion disableGutters elevation={0} square {...props} />
+))(() => ({
+  borderTop: 0,
+  borderBottom: "1px solid rgba(0, 0, 0, .125)",
+  width: "100%",
+}));
+
+const AccordionSummary = styled((props: AccordionSummaryProps) => (
+  <MuiAccordionSummary
+    expandIcon={<ExpandMoreIcon sx={{ fontSize: "2rem" }} />}
+    {...props}
+  />
+))(() => ({
+  backgroundColor: "#fcf9f2",
+  padding: "16px 0",
+  border: 0,
+  "& .MuiAccordionSummary-content": {
+    margin: 0,
+  },
+  "&:has(.Mui-expanded)": {
+    paddingBottom: 0,
+  },
+}));
+
+const AccordionDetails = styled(MuiAccordionDetails)(({ theme }) => ({
+  backgroundColor: "#fcf9f2",
+  padding: `${theme.spacing(2)} 0`,
+  border: 0,
+}));
+
+const FaqSection: React.FC = () => {
+  return (
+    <Box
+      sx={{
+        display: "flex",
+        flexDirection: "column",
+        alignItems: "center",
+        justifyContent: "center",
+        gap: 3,
+        backgroundColor: "#fcf9f2",
+        padding: "80px 24px",
+        margin: {
+          xs: "0 -16px",
+          md: 0,
+        },
+      }}
+    >
+      <Typography
+        variant="h2"
+        sx={{
+          color: "#0e0e07",
```

Comment:
For text color please do the same thing krub
MUI sx prop you can use something like this to use the theme color
`color: "text.secondary"`